src/pages/api/iexec/fund-release.tsx

```diff
@@ -0,0 +1,121 @@
+  const mongoUri = process.env.NEXT_MONGO_URI;
+  const cronSecurityKey = req.query.key;
+  const RETRY_FACTOR = 5;
+  let successCount = 0,
+    errorCount = 0;
+
+  if (cronSecurityKey !== process.env.NEXT_PRIVATE_CRON_KEY) {
+    return res.status(401).json({ message: 'Unauthorized' });
+  }
+
+  if (!chainId) {
+    return res.status(500).json('Chain Id is not set');
+  }
+
+  if (!mongoUri) {
+    return res.status(500).json('MongoDb URI is not set');
+  }
+
+  await mongoose.connect(mongoUri as string);
+
+  const platformId = process.env.NEXT_PUBLIC_PLATFORM_ID;
+
+  if (!platformId) {
+    throw new Error('Platform Id is not set');
+  }
+
+  // Check whether the user provided a timestamp or if it will come from the cron config
+  const { sinceTimestamp, cronDuration } = calculateCronData(
+    req,
+    RETRY_FACTOR,
+    EmailType.FundRelease,
+  );
+  try {
+    const response = await getNewPayment(Number(chainId), platformId, sinceTimestamp);
+    const nonSentPaymentEmails: IPayment[] = [];
+
+    // Check if some entities are not already in the DB
+    if (response.data.data.payments.length > 0) {
+      for (const payment of response.data.data.payments as IPayment[]) {
+        await checkPaymentExistenceInDb(payment, nonSentPaymentEmails, EmailType.FundRelease);
+      }
+    }
+
+    // If some entities are not already in the DB, email the hirer & persist the payment in the DB
+    if (nonSentPaymentEmails) {
+      for (const payment of nonSentPaymentEmails) {
+        let handle = '',
+          action = '',
+          address = '';
+        payment.paymentType === PaymentTypeEnum.Release
+          ? ((handle = payment.service.seller.handle),
+            (action = 'released'),
+            (address = payment.service.seller.address))
+          : ((handle = payment.service.buyer.handle),
+            (action = 'reimbursed'),
+            (address = payment.service.buyer.address));
+        console.log('Handle', handle);
+        console.log('Action', action);
+        console.log('Address', address);
+        console.log('Payment id', payment.id);
+        // Check whether the user opted for the called feature | Seller if fund release, Buyer if fund reimbursement
+        //TODO query not tested
+        const userData = await getUserWeb3mailPreferences(
+          Number(chainId),
+          platformId,
+          address,
+          Web3mailPreferences.activeOnFundRelease,
+        );
+        if (!userData?.description?.web3mailPreferences?.activeOnFundRelease) {
+          console.error(
+            `User ${address} has not opted in for the ${EmailType.FundRelease} feature`,
+          );
+          continue;
+        }
+
+        try {
+          // @dev: This function needs to be throwable to avoid persisting the entity in the DB if the email is not sent
+          await sendMailToAddresses(
+            `Funds ${action} for the service - ${payment.service.description?.title}`,
```

Comment:
- Can you try seperatly if we can already have HTML in the content please ? 
- also we need an utils function to generate the URL of a service ${platform.description.website}/services/{id}